packages/vertica-nodejs/test/integration/client/no-data-tests.js

```diff
@@ -13,7 +13,7 @@ suite.test('noData message handling', function () {
 
   var q = client.query({
     name: 'boom',
-    text: 'create table if not exists boom(id identity, size integer)',
+    text: 'create table if not exists boom(size integer)',
```

Comment:
Can we switch this to a local temp table?

Reply:
yep, thats part of the reason why I removed identity, I just forgot. Making that change